examples/tensorflow/object_detection/main.py

```diff
@@ -264,6 +272,7 @@ def evaluate(test_step, metric, test_dist_dataset, num_batches, print_freq):
     return result
 
 
+#  pylint: disable=too-many-statements
```

Comment:
Please, disable the pylint error using .pylintrc.

Reply:
@alexsu52, are you sure we should disable it globally for all NNCF files?
I'd not do it for the library part.

not sure we can easily disable it just for samples: https://github.com/palantir/python-language-server/issues/616

Reply:
@ljaljushkin Please refactor code in the object detection example then.

Reply:
refactored, pylint is happy